ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,1047 @@
+    https_port = None
+    http_port = None
+    for container in pod_obj.pod_data.get("spec", {}).get("containers", []):
+        for port_def in container.get("ports") or []:
+            name = (port_def.get("name") or "").lower()
+            container_port = port_def.get("containerPort")
+            if not container_port:
+                continue
+            if container_port == constants.OCS_METRICS_EXPORTER_HTTPS_PORT:
+                https_port = container_port
+            elif "https" in name and https_port is None:
+                https_port = container_port
```

Ruff (0.15.21)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)